specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -1284,6 +1284,40 @@
         "x-ms-odata": "#/definitions/ServicePrincipal"
       }
     },
+    "/{tenantID}/servicePrincipalsByAppId/{applicationID}/objectId": {
```

Comment:
Not a comment, just ref for me/
https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/api/functions-and-actions#servicePrincipalsByAppId